package.json

```diff
@@ -0,0 +1,106 @@
+{
+    "name": "cyberlab",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "scripts": {
+        "dev": "vite --open",
+        "test": "vitest",
+        "build": "tsc && vite build",
+        "preview": "vite preview",
+        "eslint": "eslint --ext .jsx --ext .js --ext .tsx --ext .ts --max-warnings 1 src/"
+    },
+    "dependencies": {
+        "@dnd-kit/core": "^6.1.0",
+        "@dnd-kit/sortable": "^8.0.0",
+        "@dnd-kit/utilities": "^3.2.2",
+        "@emotion/react": "^11.11.4",
+        "@emotion/styled": "^11.11.0",
+        "@fontsource/roboto": "^5.0.12",
+        "@hookform/resolvers": "^3.3.4",
+        "@mui/icons-material": "^5.15.13",
+        "@mui/lab": "^5.0.0-alpha.168",
+        "@mui/material": "^5.15.13",
+        "@mui/x-data-grid": "^6.19.6",
+        "@mui/x-date-pickers": "^6.19.7",
+        "@mui/x-tree-view": "^6.17.0",
+        "@react-pdf/renderer": "^3.4.0",
+        "@react-pdf/types": "^2.4.1",
+        "@tabler/icons-react": "^3.1.0",
+        "@tanstack/react-query": "^5.28.4",
+        "@tanstack/react-table": "^8.13.2",
+        "@tanstack/react-virtual": "^3.1.3",
+        "@tiptap/extension-link": "^2.2.4",
+        "@tiptap/extension-text": "^2.2.4",
+        "@tiptap/extension-text-align": "^2.2.4",
+        "@tiptap/extension-text-style": "^2.2.4",
+        "@tiptap/extension-underline": "^2.2.4",
+        "@tiptap/react": "^2.2.4",
+        "@tiptap/starter-kit": "^2.2.4",
+        "@visx/axis": "^3.10.1",
+        "@visx/event": "^3.3.0",
+        "@visx/group": "^3.3.0",
+        "@visx/mock-data": "^3.3.0",
+        "@visx/responsive": "^3.10.2",
+        "@visx/scale": "^3.5.0",
+        "@visx/shape": "^3.5.0",
+        "@visx/text": "^3.3.0",
+        "@visx/tooltip": "^3.3.0",
+        "axios": "^1.6.8",
+        "canvg": "^4.0.1",
+        "countries-state-list": "^1.1.0",
```

Comment:
what is this

Reply:
scrap code from earlier we may need later on